lib/Zonemaster/LDNS/RR/SPF.pm

```diff
@@ -21,6 +27,6 @@ A subclass of L<Zonemaster::LDNS::RR>, so it has all the methods of that class a
 
 =item spfdata()
 
-Returns the SPF string.
+Returns the concatenation of all the strings composing the data of the resource record.
```

Comment:
Unlike TXT resource records, there is no need to access the string list within an SPF record, because the only useful way of using SPF records is to publish SPF policies.

Reply:
I think that is true for TXT records too. Since SPF and TXT have the same RDATA format it would be less confusing if it is consistent. Since this is a breaking change, the list format could be removed from the TXT record.